documentation/README.md

```diff
@@ -1,36 +1,31 @@
-# Website
+# Documentation
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+The documentation is built using [Docusaurus 2](https://docusaurus.io/). The deployment is done through a centralized build from [IOTA WIKI](https://github.com/iota-community/iota-wiki). To run a local instance the [IOTA WIKI CLI](https://github.com/iota-community/iota-wiki-cli) is used.
 
 ## Installation
 
 ```console
-yarn install
+npm i
+npm run setup
 ```
```

Comment:
It might be worth mentioning in the README.md that `yarn` is required as well as Node version 14+?

E.g.

```md
## Prerequisites

- [Node.js v14+](https://nodejs.org/en/)
- [yarn](https://yarnpkg.com/getting-started/install)
```